Source/WebInspectorUI/UserInterface/Views/SpreadsheetStyleProperty.js

```diff
@@ -475,6 +475,27 @@ WI.SpreadsheetStyleProperty = class SpreadsheetStyleProperty extends WI.Object
         this._cssDocumentationPopover = null;
     }
 
+    // InlineSwatch delegate
+
+    inlineSwatchColorVariableCompute (swatch, variableName = null)
+    {
+        const variableNameStart = 4;
+        const varibleNameEnd = 1;
+        let computedColorCSSVariables = this.property.ownerStyle.nodeStyles.computedStyle.variablesForType(WI.CSSStyleDeclaration.VariablesGroupType.Colors);
+
+        if (!variableName)
```

Comment:
i dont think we should overload this delegate method to do more than one thing.  i think it'd be far preferred to have just one delegate method to do the basic functionality (i.e. getting the list of color variables) and then have the caller do the more advanced stuff (i.e. finding the color variable matching the given variable name)

id rework this code to be
```
inlineSwatchGetColorVariables(inlineSwatch)
{
    return this._property.ownerStyle.nodeStyles.computedStyle.variablesForType(WI.CSSStyleDeclaration.VariablesGroupType.Colors);
}
```

and then add a new function on `WI.InlineSwatch` that you'd call where needed
```
findMatchingColorVariable(variableName)
{
    let colorVariables = this._delegate?.inlineSwatchGetColorVariables?.(this) || [];

    const variableNameStart = 4;
    const variableNameEnd = 1;
    for (let colorVariable of colorVariables) {
        let index = variableName.indexOf(colorVariable.name);
        if ((!index && colorVariable.name.length === variableName.length) || (index === variableNameStart && variableNameStart + colorVariable.name.length === variableName.length - variableNameEnd))
            return colorVariable;
    }
    return null;
}
```